kernel_install: Added support for multiple kernel install files

ader1990 opened the pull request.

ader1990 committed:
kernel_install: Added support for multiple kernel install files

Commit:
Added 1 Storage TC (#274)

v-sijliu committed:
Add keywords for COREOS when check dmesg to ignore them. (#272)

* Add keywords for COREOS when check dmesg to ignore them.
Confirmed with Distro vendor.

* Remove value which may change next time

chvalean commented on Testscripts/Linux/customKernelInstall.sh:
```diff
@@ -162,8 +161,20 @@ InstallKernel()
                         prefix="localfile:"
-                        LogMsg "Installing ${CustomKernel#$prefix}"
-                        dpkg -i "${CustomKernel#$prefix}"  >> $logFolder/build-CustomKernel.txt 2>&1
+                        apt-get remove -y linux-cloud-tools-common grub-legacy-ec2
```
just apt, and add a comment on why we have to remove these packages first.

chvalean committed:
Merge pull request #264 from ader1990/hyperv_vhd_leak

hyper-v: Remove snapshots and not leak vhds

Commit:
Support tests cases against SLES15 (#232)

* Support tests cases against SLES15

* Update code based on comments

* Update code based on comments.
Fix syntax issue for BVT-CORE-TIME-CLOCKSOURCE which bring in by latest commit.

* Miss one file.

simonxiaoss committed:
Delete RunTests.ps1

This file has been renamed to Run-LisaV2.ps1.

iamshital committed:
Usage fix (#258)

* Moved Top VMs to upper part of report

* renamed:    GetSubscriptionUsage.ps1 -> Get-SubscriptionUsage.ps1

Moved from JenkinsPipelines/scripts -> Utilities
